src/Context.ts

```diff
@@ -59,3 +65,10 @@ export type SubscriptionContext = {
   log: FastifyBaseLogger;
   userId?: string;
 };
+
+export type BaseContext = Omit<Context, 'userId' | 'trackingId'>;
+
+export type AuthContext = BaseContext & {
+  userId: string;
+  trackingId: string;
+};
```

Comment:
Wouldn't this do?
```suggestion
export type AuthContext = Context;
```

Reply:
No, since Context has `userId` and `trackingId` optional and then that breaks a lot of queries/mutations where it expects for `ctx.userId` to be defined.